thermostat.cpp

```diff
@@ -577,7 +577,7 @@ bool DeRestPluginPrivate::addTaskThermostatReadWriteAttribute(TaskItem &task, ui
         {
             stream << (quint8) attrValue;
         }
-        else if (attrType == deCONZ::Zcl16BitInt || attrType == deCONZ::Zcl16BitBitMap)
+        else if (attrType == deCONZ::Zcl16BitInt || attrType == deCONZ::Zcl16BitBitMap || attrType == deCONZ::Zcl8BitBitMap)
```

Comment:
Are you sure you want to write an 8-bit bitmap as 16-bit value?!

Reply:
That was a mistake. Fixed.